src/index.js

```diff
@@ -2,12 +2,113 @@ import React from "react";
 import ReactDOM from "react-dom/client";
 import "./index.css";
 import App from "./App";
+import { EducationContainer } from "./components/EducationContainer/EducationContainer";
 import reportWebVitals from "./reportWebVitals";
+import { createBrowserRouter, RouterProvider } from "react-router-dom";
+import GlobalStoreProvider from "./globalStoreProvider";
+import sanitizeHtml from "sanitize-html"; // Import sanitize-html
+
+// Create a global function for HTML sanitization
+window.sanitizeHTML = (html) => {
```

Comment:
Sanitizing HTML seems like a complex tasks. Are there any limitations of these functions that you are aware of?

Reply:
@fissoreg I'm not aware of any limitations as of now.  But would be interested to know if there are any.